content/python/concepts/strings/terms/isdigit/isdigit.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.isdigit()'
+Description: 'Takes in a string and returns True if all characters in the string are digits and there is at least one character, False otherwise. Digits include decimal characters and digits that need special handling, such as the compatibility superscript digits. This covers digits which cannot be used to form numbers in base 10, like the Kharosthi numbers.'
```

Comment:
This description is just a copy of the python documentation, that is considered plagiarism. All docs content should be original. Please revise the description and the text on line 16 to use your own, original phrasing.

https://docs.python.org/3/library/stdtypes.html?highlight=isdigit#str.isdigit

Reply:
Hey @caupolicandiaz, I revised the description and text on line 16. (Preview below)
![image](https://github.com/Codecademy/docs/assets/58596929/5dbbd6fb-7887-43ba-9e72-7057d66f343a)